### PR TITLE
Add Keyboard Maestro v6.4.8

### PR DESCRIPTION
**Pull request by tjanson**
This is the latest minor of version 6 of [Keyboard Maestro](https://www.keyboardmaestro.com) (latest major is 7, a paid upgrade).

### CODE REVIEW

File: Casks/keyboard-maestro6.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'keyboard-maestro6' do
+  version '6.4.8'
+  sha256 '74276c1bbb2f3b6f188e3883519c6f94cca333b40dd5a38ef79af37c3d6c5c06'
+
+  # stairways.com is the official download host per the vendor homepage
+  url "https://files.stairways.com/keyboardmaestro-#{version.delete('.')}.zip"
```
Comment (vitorgalvao) [inline Casks/keyboard-maestro6.rb]: Where on the website is this, so we can verify it?

Use `#{version.no_dots}` instead.

Comment (tjanson) [inline Casks/keyboard-maestro6.rb]: There’s a link on [the Keyboard Maestro homepage](https://www.keyboardmaestro.com/) with text

> (For OS X 10.9 and older, download an old version)

which links to that host. (Also, Stairways Software Pty Ltd. is the developer of the software.)

Comment (adidalal) [inline Casks/keyboard-maestro6.rb]: http://www.keyboardmaestro.com/action/download?km-kmi-2-f from the bottom of the page is the link i found, whichh redirects to the above (ie http://files.stairways.com/keyboardmaestro-703.zip)

Comment (vitorgalvao) [inline Casks/keyboard-maestro6.rb]: > Also, Stairways Software Pty Ltd. is the developer of the software

But users won’t know that at a glance (I didn’t), so this check is still needed.

**Comment by vitorgalvao** [comment]
Looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as a0b572057bf8b7588cf0303b89b91971660927d5 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**